app/models/document.rb

```diff
@@ -126,6 +130,11 @@ def humanized_document_type
     document_type.underscore.tr("_", " ")
   end
 
+  def edition_versions
+    Version.where(item_type: "Edition", item_id: editions.select(:id))
```

Comment:
I think there's a neater way you can do this association. I've got it in my draft PR: https://github.com/alphagov/whitehall/pull/6607/files#diff-893a2b609f3fc4aade494f624a20993eb6c32598512b5e70d9ad39866b8eb51dR43

Reply:
Oh, nice! Thanks.